honeybee_energy/properties/model.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+"""Model Energy Properties."""
+from ladybug.analysisperiod import AnalysisPeriod
+
+
+class ModelEnergyProperties(object):
+    """Energy Properties for Honeybee Model.
+
+    Properties:
+        run_period
```

Comment:
I think except for `solar_distribution` the rest of the properties do not belong to Model itself. They belong to a `run` or `result-set`. This is a topic that we can talk about it during a call but for now here is how I ended up break down input / output information. This structure is modified from how OpenFOAM does structure data and is what I'm using for the latest version of radiance model folder:

1. model: geometry + metadata that describes a "simulatable" model.
2. asset: additional information that define the scene like weather files or sky in daylight model.
3. system: settings that will be used during a run.
4. output: folder for results and post-processing. 

Including run_period, simulation_outputs and time-step inside model instead of system will make your model not reusable. I have not scene the model implementation in `honeybee-core` yet but I believe this is a topic that takes a good call to be fully figured out.

Reply:
This all makes sense, @mostaphaRoudsari .  I would even say that `solar_distribution` should actually be under `system` because it s really related to what type of simulation you want to run (for example, you don't really need detailed solar distribution if you just need an estimate of total building energy use).  Model energy properties will be things like all of the unique constructions and materials across the model.  For now, I'll just keep the Model Properties as a blank object but the next PR that I send will add methods to these Model properties to search for unique constructions/materials throughout the model.